src/components/ViewRating.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+import { formatDateTime } from "@/services/utils";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck, faXmark } from "@fortawesome/free-solid-svg-icons";
+import { ProductReview } from "@/model/ProductReviewsModel";
+
+const ViewRating = ({ review }: { review: ProductReview }) => {
+  return (
+    <div className="flex justify-between px-4 mt-2 pb-2 border-b border-solid border-grey w-full">
+      <div className="w-4/12">
+        <p className="font-bold"> {review.user_id} </p>
+        <p>
+          <span className="font-bold"> Rating </span> {review.rating}
+        </p>
+      </div>
+      <div className="w-full">
+        <p>{formatDateTime(new Date(review.added_at))} </p>
```

Comment:
Use the `<time>` element: 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time